perpendicular-flap/solid-fake/fake.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import division
+
+import numpy as np
+import precice
+
+
+def displacement_param(t):
+    # this parameter exists to define a time-dependent displacement of the flap's tip
+    if t > 1:
+        return 0.5
+    else:
+        return 0.5 * t
```

Comment:
```suggestion
def displacement_param(t):
    # this parameter exists to define a time-dependent displacement of the flap's tip
    max_d = 0.5
    if t > 1:
        return max_d
    else:
        return max_d * t
```
To avoid code duplication.

You could also do `min(max_d, max_d*t)` but I personally prefer the `if...else`-way of writing things.

Reply:
I think this was somehow cached. Cannot find this piece of code anymore and is not relevant.